### PR TITLE
Convert scripts in bin to console_scripts

### PR DESCRIPTION
**Pull request by rodrigc**
See:
http://twistedmatrix.com/trac/ticket/8491

### CODE REVIEW

**Comment by alex** [comment]
looks like some of the builders need to be updated

**Comment by rodrigc** [comment]
@adiroiban: why is codecov showing 71.42% coverage here: https://codecov.io/gh/twisted/twisted/compare/8dd2292cac9be85a9f81aef336c35bdb89f46e1c...87cb6c0d9af66591d218bd53684d58e15601daf4

It looks like it is not reporting coverage of the _PY3 block.  Isn't the https://buildbot.twistedmatrix.com/builders/fedora22-py3.4-coverage build uploading coverage results to codecov.io ?

File: setup.py [hunk 1]
```diff
@@ -43,7 +43,9 @@ def main(args):
         packages=setuptools.find_packages(),
         install_requires=requirements,
         conditionalExtensions=getExtensions(),
-        scripts=getScripts(),
+        entry_points={
+               'console_scripts':  getConsoleScripts()
```
Comment (alex) [inline setup.py]: This is the wrong indentation.

File: setup3.py [hunk 2]
```diff
@@ -34,37 +34,26 @@ def find_package_modules(self, package, package_dir):
...
         cmdclass={
             'build_py': PickyBuildPy,
-            'build_scripts': PickyBuildScripts,
         },
         packages=find_packages(),
         install_requires=["zope.interface >= 4.0.2"],
         conditionalExtensions=getExtensions(),
-        scripts=getScripts(),
+        entry_points={
+               'console_scripts':  getConsoleScripts()
```
Comment (alex) [inline setup3.py]: Same here.

**Pull request merged**